### PR TITLE
367 : Performance profile and special marker for performance tests

### PR DESCRIPTION
**Pull request by mstzn36**
Performance tests are environment dependent.For this reason performance tests sometimes fail on local machines.Created maven performance profile. Perfomance profile only runs performance tests which marked PerfomanceTests marker.
PerformanceTests can be defined on class level or method level.

We can run performance tests with <code>mvn clean test -P performance</code> .

Fixes https://github.com/yegor256/takes/issues/367

### CODE REVIEW

**Comment by davvd** [comment]
@mstzn36 I will find a reviewer for your pull requests shortly, thanks for contribution!

**Comment by davvd** [comment]
@pinaf review this one, please

**Comment by pinaf** [comment]
@mstzn36 why is there an icon file in the PR?

File: src/test/java/org/takes/misc/PerformanceTests.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.takes.misc;
+
+/**
+ * Marker interface for performance tests.
+ * @author Mesut Ozen (mesutozen36@gmail.com)
+ * @version $Id$
+ * @since 1.0
```
Comment (pinaf) [inline src/test/java/org/takes/misc/PerformanceTests.java]: @mstzn36 `@since 0.24`

**Comment by pinaf** [comment]
@mstzn36 build isn't passing

**Comment by mstzn36** [comment]
@pinaf sorry about that. I pushed the icon file because i was working another issue.

**Comment by pinaf** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by davvd** [comment]
@ypshenychka please, review this task for compliance with our [quality rules](http://www.teamed.io/qa.html)

**Comment by davvd** [comment]
@rultor please deploy

**Comment by ypshenychka** [comment]
@pinaf 
According to our [QA Rules](http://www.teamed.io/qa.html):
>The code reviewer found at least three problems in the code.

Only two cosmetic issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by pinaf** [comment]
@ypshenychka I confirm

**Comment by ypshenychka** [comment]
@davvd Quality is acceptable.

**Comment by davvd** [comment]
> @davvd Quality is acceptable.

@ypshenychka thanks a lot, next time everybody should try to make it better

**Comment by davvd** [comment]
@pinaf just added 10 mins to @ypshenychka (for QA), payment ID is `64445937`... Much appreciated! **19 mins** was added to your account, payment ID `AP-5813299587122601Y`, time spent is 91 hours and 26 mins... you're getting extra minutes here (c=4)... +19 added to your rating, current score is: [+4757](http://www.netbout.com/b/35128?open=rating)

**Pull request merged**